SecurityPkg/Include/Library/Hash2CryptoLib.h

```diff
@@ -0,0 +1,30 @@
+/** @file
+  This library abstracts installation of the Hash 2 Service Binding protocol.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Update the licenses here.

Reply:
This was submitted earlier on, before forming opinions about changing these things. Will update.